bumble/hci.py

```diff
@@ -5653,6 +5716,64 @@ def __str__(self):
         )
 
 
+# -----------------------------------------------------------------------------
+class HCI_SynchronousDataPacket:
+    '''
+    See Bluetooth spec @ 5.4.2 HCI ACL Data Packets
```

Comment:
Should be `5.4.3 HCI Synchronous Data packets`

Reply:
Done